.github/workflows/chatbot.yaml

```diff
@@ -53,29 +54,25 @@ jobs:
         with:
           python-version: '3.11'
 
-      - name: Install dependencies
+      - name: Install Dependencies
         working-directory: ./recipes/natural_language_processing/chatbot
         run: make install
 
       - name: Run Functional Tests
         working-directory: ./recipes/natural_language_processing/chatbot
         run: make functional_tests
 
-      - name: Login to github container registry
-        if: >
-          (github.event_name == 'push' || github.event_name == 'schedule') &&
-          (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v'))
-        uses: docker/login-action@v3.1.0
+      - name: Login to Registry
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: redhat-actions/podman-login@v1.7
```

Comment:
Should we have the name redhat in an upstream project?

Reply:
It is an open source GitHub Action https://github.com/redhat-actions/podman-login.

Reply:
Ok, SGTM